cmd/ido2icingadb/README.md

```diff
@@ -0,0 +1,120 @@
+# IDO2IcingaDB
+
+![Icinga Logo]
+
+#### Table of Contents
+
+- [About](#about)
+- [License](#license)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Support](#support)
+- [Contributing](#contributing)
+
+## About
+
+This commandline tool migrates history data from [IDO] to [Icinga DB].
+Or, more precisely: from the IDO SQL database to the Icinga DB one.
+
+!!! info
+
+    Everything else is already populated by Icinga DB itself.
+    Only the past history data of existing IDO setups
+    isn't known to Icinga DB without migration from IDO.
+
+## License
+
+This tool and its documentation are licensed under the terms
+of the GNU General Public License Version 2,
+you will find a copy of this license in the [LICENSE] file.
+
+## Installation
+
+1. Add the official release repository as described
+   in the [installation chapter] of the Icinga DB documentation.
+2. Install the package `ido2icingadb`.
+
+## Usage
+
+### Icinga DB
+
+1. Make sure Icinga DB is up, running and writing to its database.
+2. Optionally disable Icinga 2's IDO feature.
+
+!!! warning
+
+    Disable the IDO feature – the sooner, the better!
+    There will be duplicate events for the period both IDO and Icinga DB are active.
+
+At best both enable Icinga DB and disable IDO within one Icinga 2 reload.
+
+### Configuration file
+
+Create a YAML file like this somewhere:
+
+```yaml
+icinga2:
+   # Content of /var/lib/icinga2/icingadb.env
+   env: "da39a3ee5e6b4b0d3255bfef95601890afd80709"
+   # Name of the main Icinga 2 endpoint writing to IDO
+   endpoint: master-1
+# IDO database
+ido:
+   type: pgsql
+   host: 192.0.2.1
+   port: 5432
+   database: ido
+   user: ido2icingadb
+   password: a23456
+# Icinga DB database
+icingadb:
+   type: mysql
+   host: 2001:db8::192.0.2.2
+   port: 3306
+   database: icingadb
+   user: ido2icingadb
+   password: fd3e4aa3e0d5c833dd17f5c157f9a9d2
+```
+
+### Cache directory
+
+Choose a (not necessarily yet existing) directory for IDO2IcingaDB's
+internal cache. If either there isn't much to migrate or the migration
+process isn't expected to be interrupted by a reboot (of the machine
+IDO2IcingaDB runs on), `mktemp -d` is enough.
```

Comment:
Typically? _Typical_ users of this tool have a dedicated "never" rebooting DB server with all DBs incl. IDO+IDB and enough stuff to migrate to run this tool on that machine to reduce latency.

Reply:
And if you're unlucky, it is managed by a different team which won't give you shell access to that server.

Other suggestion (doesn't matter what you consider typical use): by default, don't require `-t` but rather use a temporary directory that is fully managed by the tool (i.e. always cleaned up, also in case of errors). The documentation could then be rewritten to say that there is an internal cache and that you can manually specify a path if needed (don't rebuild on restart, space constraints in `/tmp`, etc.).